src/test/end-to-end/tests/system-admin/user-management-test.ts

```diff
@@ -25,19 +25,7 @@ Scenario('I as a system admin should be able to update a users role and delete a
     I.waitForText(testEmail);
     I.click('#manage-link');
     I.waitForText('Manage ' + testEmail);
-    I.click('Change');
```

Comment:
The text for the above test needs changing to remove 'update a users role'

Reply:
Thanks updated the title